tests/providers/apache/hdfs/hooks/test_hdfs.py

```diff
@@ -29,9 +31,7 @@
     snakebite_loaded = True
 except ImportError:
     snakebite_loaded = False
-
-if not snakebite_loaded:
-    raise unittest.SkipTest("Skipping test because HDFSHook is not installed")
+    pytestmark = pytest.mark.skip("Skipping test because HDFSHook is not installed")
```

Comment:
Hmm, why are you defining `pytestmark`? It's not being used?

Reply:
Ah TIL, it's a global var, didn't know that.

Reply:
Yeah. learn about it today too :)